os-server/src/main/java/app/openschool/course/module/quiz/QuizController.java

```diff
@@ -0,0 +1,183 @@
+package app.openschool.course.module.quiz;
+
+import app.openschool.common.response.ResponseMessage;
+import app.openschool.course.module.quiz.api.dto.CreateQuizDto;
+import app.openschool.course.module.quiz.api.dto.EnrolledQuizAssessmentRequestDto;
+import app.openschool.course.module.quiz.api.dto.EnrolledQuizAssessmentResponseDto;
+import app.openschool.course.module.quiz.api.dto.ModifyQuizDataRequest;
+import app.openschool.course.module.quiz.api.dto.QuizDto;
+import app.openschool.course.module.quiz.api.mapper.QuizMapper;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import java.util.Locale;
+import javax.validation.Valid;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+
+
+@RestController
+@RequestMapping("/api/v1/quizzes")
```

Comment:
as the quizzes only available for modules i think the path to access quizzes can be following:
/api/v1/modules/{moduleId}/quizzes